psi4/src/psi4/libmints/eri.cc

```diff
@@ -179,10 +179,31 @@ Libint2ERI::Libint2ERI(const IntegralFactory *integral, double screening_thresho
                              std::max(basis3()->max_nprimitive(), basis4()->max_nprimitive()));
     const auto max_precision = std::numeric_limits<double>::epsilon() * 1e-13;
     const auto engine_precision = screening_threshold == 0.0 ? max_precision : std::numeric_limits<double>::epsilon() * std::pow((double)max_nprim, -4.0);
+
+    bool dummy1 = basis1()->l2_shell(0) == libint2::Shell::unit();
+    bool dummy2 = basis2()->l2_shell(0) == libint2::Shell::unit();
+    bool dummy3 = basis3()->l2_shell(0) == libint2::Shell::unit();
+    bool dummy4 = basis4()->l2_shell(0) == libint2::Shell::unit();
+
+     if (!dummy1 && !dummy2 && !dummy3 && !dummy4) {
+        braket_ = libint2::BraKet::xx_xx;
+    } else if (!dummy1 && dummy2 && !dummy3 && !dummy4) {
+        braket_ = libint2::BraKet::xs_xx;
+    } else if (!dummy1 && !dummy2 && !dummy3 && dummy4) {
+        braket_ = libint2::BraKet::xx_xs;
+    } else if (!dummy1 && dummy2 && !dummy3 && dummy4) {
+        braket_ = libint2::BraKet::xs_xs;
+    } else {
+        throw PSIEXCEPTION("Bad BraKet type in Libint2TwoElectronInt");
+    }
```

Comment:
This block doesn't hurt, but I don't think it's necessary - the base class's `common_init` should have been called by this time, so `braket_` should already be set by now.